app/src/main/res/values/strings.xml

```diff
@@ -429,4 +429,5 @@
   <string name="coming_soon">Coming Soon</string>
   <string name="recommended_stories">Recommended Stories</string>
   <string name="stories_for_you">Stories For You</string>
+  <string name="label_app_version">App Version</string>
```

Comment:
Add this line below
https://github.com/oppia/oppia-android/blob/2d8930664b86a98b16ec1d0d98c6f2742ce13baa/app/src/main/res/values/strings.xml#L332

Similar to 
https://github.com/oppia/oppia-android/blob/2d8930664b86a98b16ec1d0d98c6f2742ce13baa/app/src/main/res/values/strings.xml#L302-L303